test/chemistry/circuit/__init__.py

```diff
@@ -0,0 +1,11 @@
+# This code is part of Qiskit.
```

Comment:
This PR is also changing/adding to chemistry test folder? I am not sure that these package files are really needed there maybe @manoelmarques  can comment

Reply:
I noticed that some ML tests were not being executed since the subfolders missed the init file. I then looked at all the other folders and noticed folders in Chemistry were missing init files too and tests weren't being executed, so I added init files to all folders that were missing it. I can go back however and create a PR just for the init folders in Chemistry if ML will be separated.